inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/model/Viewport.java

```diff
@@ -15,46 +15,47 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package de.tudarmstadt.ukp.inception.experimental.api.message;
+package de.tudarmstadt.ukp.inception.experimental.api.model;
 
 import java.util.List;
 
-import de.tudarmstadt.ukp.inception.experimental.api.Annotation;
-
-public class DocumentMessage
+public class Viewport
 {
-    private int document;
-    private Character[] viewportText;
-    private List<Annotation> annotations;
+    private List<List<Integer>> viewport;
+    private List<String> disabledLayers;
 
-    public int getDocument()
+    public Viewport()
     {
-        return document;
     }
 
-    public void setDocument(int aDocument)
+    public Viewport(List<List<Integer>> aViewport)
     {
-        document = aDocument;
+        viewport = aViewport;
     }
 
-    public Character[] getViewportText()
+    public Viewport(List<List<Integer>> aViewport, List<String> aDisabledLayers)
     {
-        return viewportText;
+        viewport = aViewport;
+        disabledLayers = aDisabledLayers;
     }
 
-    public void setViewportText(Character[] aViewportText)
+    public List<List<Integer>> getViewport()
     {
-        this.viewportText = aViewportText;
+        return viewport;
     }
 
-    public List<Annotation> getAnnotations()
+    public void setViewport(List<List<Integer>> aViewport)
     {
-        return annotations;
+        viewport = aViewport;
     }
 
-    public void setAnnotations(List<Annotation> aAnnotations)
+    public List<String> getDisabledLayers()
     {
-        annotations = aAnnotations;
+        return disabledLayers;
     }
 
+    public void setDisabledLayers(List<String> aDisabledLayers)
```

Comment:
Why inverse logic of disabling certain layers? Why not providing a list of the layers for which we want to receive events?